pyproject.toml

```diff
@@ -0,0 +1,132 @@
+# this file contains:
+# 1 build system configuration
+# 2 project configuration
+# 3 tool configuration, for:
+# - hatch
+# - black
+# - ruff
+# - pytest
+# - mypy
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "sphinxext-altair"
+authors = [ {name = "sphinxext-altair Contributors"} ]
+dependencies = [
+    "sphinx",
+    "docutils",
+    "altair>=5.0.0",
```

Comment:
Looks great @binste! One question, why it requires altair >= 5?

Reply:
Fair point, I thought it's sort of a clean break that prior to 5 one would use the extension from within Altair but actually this also works for version 4. Tested it and lowered the requirement. Will merge once tests passed.